core/src/main/java/org/radargun/stages/test/legacy/TimeStressorCompletion.java

```diff
@@ -12,6 +12,7 @@
  */
 public class TimeStressorCompletion extends AbstractCompletion {
 
+   protected static final String PROGRESS_STRING = "Number of operations executed by this thread: %d. Elapsed time: %s. Remaining: %s. Total: %s.";
```

Comment:
Because this `PROGRESS_STRING` reports the time elapsed and duration while the one in `CountStressorCompletion` reports executed and total operation count.